src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/FreeText.java

```diff
@@ -0,0 +1,53 @@
+package uk.nhs.digital.nhsconnect.nhais.model.edifact;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.message.EdifactValidationException;
+
+/**
+ * Example FTX+RGI+++WRONG HA - TRY SURREY'
+ */
+@RequiredArgsConstructor
+@Getter
+public class FreeText extends Segment {
+    private static final String KEY = "FTX";
+    private static final String QUALIFIER = "RGI";
+    public static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+
+    private final String textLiteral;
+
+    public static FreeText fromString(String edifactString) {
+        if (!edifactString.startsWith(FreeText.KEY_QUALIFIER)) {
+            throw new IllegalArgumentException("Can't create " + FreeText.class.getSimpleName() + " from " + edifactString);
+        }
+        String[] split = edifactString.split("'")[0].split("\\+");
+        return new FreeText(split[4]);
+    }
+
+    @Override
+    public String getKey() {
+        return KEY;
+    }
+
+    @Override
+    public String getValue() {
+        return String.join(PLUS_SEPARATOR,
```

Comment:
value starts with "+" ?

Reply:
`join` function 1st param is the string to be used as "joiner"

Reply:
My bad! :)